### ISSUE TITLE
Question about Geo-registration

### ISSUE THREAD

**Issue by LEEJUHUI6635** [colmap/colmap#1738]
Firstly, thank you for your nice research. I have a question about geo-registration of colmap.

Without a GPS sensor, I obtained an odometry(camera to world coordinate) in metric units through VINS-Fusion(IMU + camera) algorithm. What I know is the pose of the camera to world coordinate in metric units through VIO algorithm. Should I put this translation in the geo-registration text file right away?

I understood that the world coordinate system in the colmap is different from the world coordinate system in metric coordinate(using VIO algorithm). I couldn’t understand how to calculate the similarity transformation only with the translation of the camera to world coordinate.

Thanks once again.

**Comment by ahojnnes** [comment]
Yes, you can directly enter the VIO translation + rotation and colmap will find an "optimal" 3d similarity to align the whole trajectory. The rotation part is currently unused in this alignment procedure though.

**Comment by junfish** [comment]
Thanks for your contribution to the research community @ahojnnes . One more question. Does the scale of world coordinates matter? Or say, am I at liberty to adjust the scale to suit our specific requirements?
